packages/arb-avm-cpp/tests/arbcore.cpp

```diff
@@ -58,16 +58,15 @@ void waitForDelivery(std::shared_ptr<ArbCore>& arbCore) {
     ArbCore::message_status_enum status;
     while (true) {
         status = arbCore->messagesStatus();
-        if (status != ArbCore::MESSAGES_EMPTY &&
-            status != ArbCore::MESSAGES_READY) {
+        if (status != ArbCore::MESSAGES_READY) {
             break;
```

Comment:
Won't this return too early on `MESSAGES_LOADING`? I think we should specifically check that `status == ArbCore::MESSAGES_EMPTY`.

Reply:
:+1: 
Also check for MESSAGES_ERROR